internal/k8s/apply.go

```diff
@@ -6,19 +6,23 @@ import (
 	"fmt"
 	"io"
 	"os/exec"
+
+	"github.com/windmilleng/tilt/internal/logger"
 )
 
-func Apply(ctx context.Context, rawYAML string, stdout io.Writer, stderr io.Writer) error {
+func Apply(ctx context.Context, rawYAML string) error {
 	// TODO(dmiller) validate that the string is YAML and give a good error
 	c := exec.CommandContext(ctx, "kubectl", "apply", "-f", "-")
 	r := bytes.NewReader([]byte(rawYAML))
 	c.Stdin = r
 
-	c.Stdout = stdout
+	writer := logger.Get(ctx).Writer(logger.VerboseLvl)
```

Comment:
this seems like something we should display to the user by default?

Reply:
sounds good